database/secret_template.yaml

```diff
@@ -13,6 +13,7 @@ objects:
       app: topological-inventory
   stringData:
     database-password: "${DATABASE_PASSWORD}"
```

Comment:
Do you think we still need this split out as an extra entry? Or can we remove this here as well?